pallets/settlement/src/lib.rs

```diff
@@ -1364,29 +1331,44 @@ impl<T: Trait> Module<T> {
         }
     }
 
-    fn is_instruction_executed(
+    /// Schedule a given instruction to be executed on the next block only if the
+    /// settlement type is `SettleOnAffirmation` and no. of affirms pending is 0.
+    fn restricted_schedule_instruction(
```

Comment:
```suggestion
    fn maybe_schedule_instruction(
```
For consistency with other pallets.